.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react",
+    ],
+    "plugins": [
+      "@babel/plugin-syntax-dynamic-import",
+      "@babel/plugin-proposal-class-properties",
+    ]
+  }
+  
```

Comment:
missing line here but otherwise everything is okay.

Reply:
cool thanks i did not see that, let me fix it